scene/gui/tab_container.cpp

```diff
@@ -417,7 +419,13 @@ void TabContainer::set_current_tab(int p_current) {
 	}
 
 	_change_notify("current_tab");
-	emit_signal("tab_changed", current);
+	if (pending_previous == current)
+		emit_signal("tab_selected", current);
```

Comment:
I don't understand. What I mean is that:
```
	if (pending_previous == current)
		emit_signal("tab_selected", current);
	else {
		previous = pending_previous;
		emit_signal("tab_selected", current);
		emit_signal("tab_changed", current);
	}
```
should be the same as
```
	emit_signal("tab_selected", current);
	if (pending_previous != current) {
		previous = pending_previous;
		emit_signal("tab_changed", current);
	}
```
which is IMO much cleaner. Or does the emission of `tab_selected` change the value of either `pending_previous` or `current`?